src/main/target/STM32F7X2/target.h

```diff
@@ -23,27 +23,32 @@
 
 #define USBD_PRODUCT_STRING     "S7X2"
 
+#define USE_BEEPER
+
 #define DEFAULT_VOLTAGE_METER_SOURCE VOLTAGE_METER_ADC
 
-// MPU6500 interrupt
+// MPU interrupt
 #define USE_EXTI
 #define USE_MPU_DATA_READY_SIGNAL
 //#define DEBUG_MPU_DATA_READY_INTERRUPT
+#define MPU_INT_EXTI PB15 // XXX Should be gone
 
 #define USE_ACC
-#define USE_FAKE_ACC
-#define USE_ACC_MPU6050
-#define ACC_MPU6050_ALIGN CW270_DEG
-
 #define USE_GYRO
+
+#define USE_FAKE_ACC
```

Comment:
Do we need to keep the `FAKE_` drivers in here? I think this will lead to users loading a wrong config, and then trying to fly it, which will not be stopped because the `FAKE_` devices are recognised.